devtools/prettier.py

```diff
@@ -3,6 +3,11 @@
 from collections import OrderedDict
 from collections.abc import Generator
 
+try:
+    from sqlalchemy.ext.declarative import DeclarativeMeta
+except ImportError:
+    DeclarativeMeta = None
```

Comment:
Seems like there's a preference for lazy imports in this library, maybe so that importing devtools without using it is inexpensive. This should probably be moved to where it's used.

Also `DeclarativeMeta = ()` would allow it to be placed in the list with everything else instead of writing `if DeclarativeMeta is not None:`, since `isinstance(x, ())` is always false. But maybe that's less readable.

Reply:
Actually, the best option is probably to make a new metaclass with an `__instancecheck__` like `MetaDataClassType` and `MetaLaxMapping` so that other values don't have to check `or isinstance(value.__class__, t)` a bunch of times.